validator/accounts/testing/mock.go

```diff
@@ -29,6 +29,8 @@ type Wallet struct {
 	UnlockAccounts    bool
 	lock              sync.RWMutex
 	HasWriteFileError bool
+	WalletDir         string
```

Comment:
Is the word `Wallet` needed in `WalletDir`?
We already know it's a wallet, and writing `wallet.Dir` is more natural than `wallet.WalletDir`.

Reply:
yeah good point i will change this I think I was just following naming of other parts of the code